gricleaner.py

```diff
@@ -76,22 +76,31 @@ def get_digest(self, repo, tag):
         response = requests.head(self.registry + path, headers=headers, verify=self.requests_verify)
         return response.headers["Docker-Content-Digest"]
 
-    def delete_image(self, repo, tag):
+    def delete_image(self, repo, tag, purge):
```

Comment:
please add default value for `purge`, making it api compatible and having the parameter optiona.